gulpfile.js

```diff
@@ -38,6 +39,9 @@ let zaSfxExe = path.join(zaRoot, '7zS.exe');
 let zaExtra7z = path.join(zaRoot, '7z920_extra.7z');
 let rhZip = path.join(zaRoot, 'resource_hacker.zip');
 let rhExe = path.join(zaRoot, 'ResourceHacker.exe');
+let zaElectronPackage = path.join(zaRoot, artifactName + '-win32-x64');
+let bundled7z = path.join(zaRoot, artifactName +'-win32-x64.7z');
+let installerExe = path.join(zaRoot, artifactName + '-' + pjson.version + artifactType + '-installer.exe');

```

Comment:
Declaring installerExe here means that artifactType will always be empty, hence both simple and bundled installers will have the same name.